azure-pipelines/end-to-end-tests-dir/artifacts.ps1

```diff
@@ -210,7 +210,7 @@ try {
     #  can't deactivate without postscript:
     $output = Run-VcpkgAndCaptureOutput -ForceExe deactivate
     Throw-IfNotFailed
-    Test-Match $output "no postscript file: rerun with the vcpkg shell function rather than executable"
+    Test-Match $output "no postscript file: rerun with the same arguments as vcpkg-shell"
```

Comment:
```suggestion
    Test-Match $output "no postscript file: run vcpkg-shell with the same arguments"
```
Consider this wording; no change requested if you don't think it's more understandable.

Reply:
I think your suggestion is an improvement